packages/core/src/routes/admin-user.ts

```diff
@@ -50,4 +50,28 @@ export default function adminUserRoutes<T extends AuthedRouter>(router: T) {
       return next();
     }
   );
+
+  router.patch(
+    '/user/:userId/custom-data',
+    koaGuard({
+      params: object({ userId: string().min(1) }),
+      body: customDataGuard,
```

Comment:
in a PATCH request, we usually use `{ customData: {} }`

Reply:
both the code & the PR summary is updated